bitnami/logstash/templates/sts.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "logstash.fullname" . }}
+  labels: {{- include "logstash.labels" . | nindent 4 }}
+spec:
+  serviceName: {{ include "logstash.fullname" . }}-headless
+  replicas: {{ .Values.replicaCount }}
+  podManagementPolicy: {{ .Values.podManagementPolicy }}
+  updateStrategy:
+    type: {{ .Values.updateStrategy }}
+    {{- if (eq "OnDelete" .Values.updateStrategy) }}
+    rollingUpdate: null
+    {{- end }}
+  selector:
+    matchLabels: {{- include "logstash.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: {{- include "logstash.labels" . | nindent 8 }}
+      annotations:
+        checksum/configuration: {{ include (print $.Template.BasePath "/configuration-cm.yaml") . | sha256sum }}
+    spec:
+{{- include "logstash.imagePullSecrets" . | nindent 6 }}
```

Comment:
I would indent this so it is under spec

Reply:
We can discus this in other moment. No other chart indent this field.